### PR TITLE
[Hub Generated] Review request for Microsoft.Billing to add version preview/2018-11-01-preview

### PR DESCRIPTION
**Pull request by batman11235**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/batman11235/azure-rest-api-specs/dev-billing-Microsoft.Billing-2018-11-01-preview...master/:tab?).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/6396 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/6396/commits/cf693ec48060525d7c36c4eafe8857fc26f31ca3

**Comment by nschonni** [comment]
typo: Insufficienct -> Insufficient

File: specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json [hunk 1]
```diff
@@ -3735,9 +3739,64 @@
...
           "type": "string",
           "description": "The destination invoice section id."
+        },
+        "destinationBillingProfileId": {
+          "type": "string",
+          "description": "The destination billing profile id."
+        }
+      }
+    },
+    "ValidateSubscriptionTransferEligibilityResult": {
```
Comment (RyanBensonMSFT) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json]: These new types aren't used anywhere in your APIs. What was your intention with these types?

Comment (batman11235) [reply specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json]: Fixed it, sorry forgot to copy the eligibility APIs from our local json files

File: specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json [hunk 2]
```diff
@@ -4738,6 +4893,58 @@
...
+        }
+      }
+    },
+    "ProductTransferValidationErrorCode": {
+      "type": "string",
+      "description": "Error code of the transfer validation response.",
+      "enum": [
+        "InvalidSource",
+        "ProductNotActive",
+        "InsufficienctPermissionOnSource",
```
Comment (nschonni) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json]:
```suggestion
        "InsufficientPermissionOnSource",
```
Insufficient

File: specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json [hunk 3]
```diff
@@ -4738,6 +4893,58 @@
...
+        "InsufficienctPermissionOnDestination",
```
Comment (nschonni) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json]:
```suggestion
        "InsufficientPermissionOnDestination",
```

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Review by wilcobmsft — APPROVED** [review]

**Review by praries880 — APPROVED** [review]

**Pull request merged**